### PR TITLE
Adds 2 (TWO) guns to wizard's spellbook

### PR DESCRIPTION
**Pull request by Arzion5**
:cl:
add: Added a stetchkin gun and a shotgun gun to wizard's spellbook. Not every situation can be solved with magic.
/:cl:

Fun gimmicky item. [Reference](https://www.youtube.com/watch?v=4cZqRzHnI8s)

### CODE REVIEW

**Comment by ghost** [comment]
I'd honestly prefer a sawed off shotgun for wizards:

**Comment by PopNotes** [comment]
I think it would actually be better if wizards summoned shotties instead of surplus rifles

**Comment by GuyonBroadway** [comment]
Honestly I would rather see Summon guns made into something that summons a fully loaded sketchkin or shotgun or something on a fairly long CD.

**Review by GuyonBroadway — DISMISSED** [review]
code is OK

**Comment by PopNotes** [comment]
PR is bad tbh, nobody's going to spend precious wizziepoints on individual guns, make them summonable like with the surplus rifles

**Comment by GuyonBroadway** [comment]
This has actually inspired me to make a better summon guns PR.

**Comment by GuyonBroadway** [comment]
The code is TECHNICALLY fine but it needs some input from headmins.

File: hippiestation/code/game/gamemodes/wizard/spellbook.dm [hunk 1]
```diff
@@ -300,6 +300,18 @@ F
 /obj/item/spellbook/oneuse/smoke/lesser //Chaplain smoke book
 	spell = /obj/effect/proc_holder/spell/targeted/smoke/lesser
 
+/datum/spellbook_entry/item/stetchkin_gun
+	name = "Stetchkin Gun"
+	desc = "'Sometimes you just need a little less gun.'"
```
Comment (KayeArray) [inline hippiestation/code/game/gamemodes/wizard/spellbook.dm]: what's with these triple quotes?

File: hippiestation/code/game/gamemodes/wizard/spellbook.dm [hunk 2]
```diff
@@ -300,6 +300,18 @@ F
...
+	item_path = /obj/item/gun/ballistic/automatic/pistol
+	cost = 1
+
+/datum/spellbook_entry/item/shotgun_gun
+	name = "Shotgun Gun"
+	desc = "'The answer? Use a gun.'"
```
Comment (KayeArray) [inline hippiestation/code/game/gamemodes/wizard/spellbook.dm]: same here

File: hippiestation/code/game/gamemodes/wizard/spellbook.dm [hunk 3]
```diff
@@ -300,6 +300,18 @@ F
...
+	item_path = /obj/item/gun/ballistic/shotgun //'And if that don't work? Use more gun.'
```
Comment (KayeArray) [inline hippiestation/code/game/gamemodes/wizard/spellbook.dm]: you might want to give them a slightly better shotgun than the generic one that is basically never seen

**Comment by tortellinitony** [comment]
Why do you want the wizard to have guns? The wizard uses magic. He has a spell that summons guns, but he doesn't buy guns outright. This is retarded. STOP

**Comment by ghost** [comment]
shotgun wizards are funny though

similar to muscle wizards

**Comment by tortellinitony** [comment]
I don't see how a wizard having a gun is inherently "funny," all I think it does is go against how the wizard antagonist behaves

**Comment by PopNotes** [comment]
Subversion of expectations is a root form of comedy

**Comment by tortellinitony** [comment]
No, that's retarded. You can't justify saying something is funny because you didn't expect it. This is a terrible idea. Wizards use magic, not guns.

**Comment by PopNotes** [comment]
I'm just letting you know because you have as much of a sense of humor as a US politician has a sense of right and wrong.

**Comment by tortellinitony** [comment]
Oh boy, I have no sense of humor because I don't like adding dumb shit with the excuse that "it's funny."

**Comment by PopNotes** [comment]
And yet we've had farting and butts for how long now?

**Comment by tortellinitony** [comment]
Longer than I've been here, so I don't see how that supports your argument since I couldn't have objected to them being added.

**Comment by tortellinitony** [comment]
At this point, the butt stuff has become an accepted part of "Hippie features." I don't think that "the wizard has guns" would ever become a respected feature associated with Hippie Station. I'd rather people see our wizard gamemode as having unique and well-thought-out spells, anyway.

**Comment by GuyonBroadway** [comment]
Just buying a gun in silly, if he had a summon guns spell that called a fresh loaded weapon to his hands (lesser summon guns is rather poo) that would be more "gun wizardy"

**Comment by Spankmaster1066** [comment]
Yea, it's a bit odd but I don't mind it. It  would make more sense if it summoned a stetchkin with one mag like every 5 mins or so. I dono.

**Comment by Arzion5** [comment]
I'd appreciate if someone made a spell that gives you a spell that summons one instance of the respective guns. Or I can do it later

**Comment by KayeArray** [comment]
how about a spell that fires bullets instead.

**Comment by Arzion5** [comment]
I have no idea how to do that

**Comment by GuyonBroadway** [comment]
Look at how fireball works and switch out the fireball for a bullet.

**Comment by Arzion5** [comment]
I'll try

**Comment by GuyonBroadway** [comment]
However for the meme of "Summon guns" it would be better just to create a new shotgun (most guns come pre-loaded) and put it in the wizards hands.

However I would make a brand new PR for this purpose.

**Comment by Arzion5** [comment]
Combat shotgun summon spell for 2 spell points, but I am not sure if they come pre-loaded with lethal shells

**Comment by GuyonBroadway** [comment]
If they don't you can make a child of combat shotguns that does.

**Comment by ghost** [comment]
Probably better to have this as a new spell that summons a weapon in hand or something.

**Comment by ghost** [comment]
Stale.

**Pull request closed without merging**